src/v3/src/transformer/phone/phoneEnrollmentTransformer.ts

```diff
@@ -102,9 +114,20 @@ export const transformPhoneEnrollment: IdxStepTransformer = ({ formBag, transact
 
   uischema.elements = [
     titleElement,
-    stepper,
   ];
 
+  const phoneMethodOptions = methodTypeElement!.options!.inputMeta.options!;
+  const firstOptionMethod = phoneMethodOptions[0].value;
+  if (phoneMethodOptions.length === 1) {
+    if (firstOptionMethod === 'sms') {
+      uischema.elements = uischema.elements.concat(smsOptionElements);
+    } else {
+      uischema.elements = uischema.elements.concat(voiceOptionElements);
+    }
+  } else {
+    uischema.elements.push(bothOptionStepper);
+  }
```

Comment:
To take this even further, optionally we could inline the array of elements (i.e. smsOptionElements and voiceOptionElements) inside the conditionals rather than defining them in the upper scope.